core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -654,6 +654,64 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(7, consumer.committed(Set(tp2).asJava).get(tp2).offset)
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testCommitAsyncCompletedConsumerCloses(quorum: String, groupProtocol: String): Unit = {
```

Comment:
We have a new `PlainTextConsumerCommitTest` for all commit-relates tests. These 2 should go there I would say.

Reply:
nit: maybe better name testCommitAsyncCompleted**Before**ConsumerCloses (clearer and consistent with the similar one below)

Reply:
Done